src/translator/logistic_regressor_quality_estimator.h

```diff
@@ -0,0 +1,70 @@
+#pragma once
+
+#include <vector>
+
+#include "definitions.h"
+#include "iquality_estimator.h"
+#include "matrix.h"
+
+namespace marian::bergamot {
+// ASCII and Unicode text files never start with the following 64 bits
+constexpr std::size_t BINARY_QE_MODEL_MAGIC = 0x78cc336f1d54b180;
+/// The current Quality Estimator model is a Logistic Model implemented through
+/// a linear regressor + sigmoid function. Simply speaking, an LR model depends
+/// on features to be scaled, so it contains four elements of data: a vector of coefficients
+/// and an intercept (which represents the linear model) and a vector of means and stds
+/// (which are necessary for feature scaling).
+///
+/// These variables are firstly initialized by parsing a file (which comes from memory),
+/// and then they are used to build a model representation
+///
+class LogisticRegressorQualityEstimator : public IQualityEstimator {
+  friend class LogisticRegressorQualityEstimatorTest;
```

Comment:
We removed the `friend` class and we also changed the `qualtiy_estimator_tests` to test `computeQualityScores` method instead of private one. Commit 33e6a87